Userland/Libraries/LibAudio/Loader.h

```diff
@@ -32,7 +39,13 @@ class LoaderPlugin {
     explicit LoaderPlugin(NonnullOwnPtr<SeekableStream> stream);
     virtual ~LoaderPlugin() = default;
 
-    virtual LoaderSamples get_more_samples(size_t max_bytes_to_read_from_input = 128 * KiB) = 0;
+    // Load as many audio chunks as necessary to get up to the required samples.
+    // A chunk can be anything that is convenient for the plugin to load in one go without requiring to move samples around different buffers.
+    // For example: A FLAC, MP3 or QOA frame.
+    // The chunks are returned in a vector, so the loader can simply chunks as long as the sample amount was not reached yet.
```

Comment:
Missing verb `... can simply [add] chunks, [until] the [requested] sample amount [is reached].`?